src/main.cxx

```diff
@@ -0,0 +1,11 @@
+#include <vector>
+
+import Image;
+
+int main() {
+    Image test {3, 3, 2, 255};
+    std::vector<int> pixels {255, 0, 0, 0, 255, 0, 0, 0, 255, 255, 255, 0, 255, 255, 255, 0, 0, 0};
```

Comment:
I would expose in the Image library a pixel type.

Reply:
Sorry for the late reply, what does expose mean?